initializer.sh

```diff
@@ -147,5 +146,10 @@ cd ~/.radicle/seed/creature-pigeon
 
 echo "setup.sh script is completed!"
 
+echo "starting up httpd for web UI"
+cd /home/seed/radicle-explorer
+export LD_LIBRARY_PATH=/usr/lib:$LD_LIBRARY_PATH
+/home/seed/.radicle-httpd/bin/radicle-httpd & /root/.bun/bin/bun start
+
 # Keep the container to run all the time
-sleep infinity
+#sleep infinity
```

Comment:
Since we had `/home/seed/.radicle-httpd/bin/radicle-httpd & /root/.bun/bin/bun start` running in the background, `sleep infinity` is not needed anymore right?

Reply:
Yeah that's right the last command that launches `bun run start` should stay on foreground. `radicle-httpd` is on background.